First draft of selection-process.md (#960)
web-platform-tests/interop: 2026/selection-process.md

Interop 2026, the timeline and the selection process is as described
below.

## Timeline

This is the general timeline for the Interop 2026 proposal selection 
process. The dates are centered on the Interop team meeting (Thursdays).
Please note that the timelines are subject to change.

*   Proposal submission window (~3 weeks): Sept 18th, 2025 to Oct 8th, 2025
*   Proposal selection: Before December 18th 2025
*   Scope of the Interop 2026 project published: the first half of February, 2026.

## Proposal selection process

The aim of the proposal selection process is to generate a shared
understanding of the merits of each proposal, and converge on a
decision about the most impactful areas in which we can improve
interop on the web over 2026.

As in Interop 2024 and 2025, details of proposal selection and ranking
remain participant confidential throughout the process. Final decisions
will be published as the Interop team.

Confidentiality does not prevent participating organizations from

> sharing their own priorities or roadmaps outside of the Interop program.

Reviewer: After we launched Interop 2025 I commented on some proposals to give additional feedback, for example in https://github.com/web-platform-tests/interop/issues/755#issuecomment-2674354422.

I wasn't sure if commenting about our plans for https://crbug.com/40768696 on the proposal itself would be OK, so I'd like to clarify that here. The simplest edit I'd like to suggest is to remove the "outside the Interop program" part:

```suggestion
sharing their own priorities or roadmaps.
```

In other words, it's OK to share such information both during the process and to follow up later to say that a bug has been fixed or a feature shipped, for example. This information is often discoverable from a bug or something linked from the proposal anyway.

Reviewer: I understand the concern, and think it comes down to the expectation that positions in the proposal selection process aren't revealed with a wink and a nudge. In past years we've proposed that confidentiality shouldn't apply to our own Interop positions, and that's still a change I'd welcome, but that's not the intention here.

Wording is hard, but the best I've been able to come up with is something like this:

> Confidentiality does not prevent participating organizations from sharing their own priorities or roadmaps, as distinct from positions in the proposal selection process.

I'd be OK with or without the extra clause, to me it's clear in either case that trying to find loopholes for confidentiality will not be well received and would be a breach of trust.

Reviewer: This was discussed in the meeting today. @jgraham preferred the additional clause, so as an edit it would be this:

```suggestion
sharing their own priorities or roadmaps, as distinct from positions in the
proposal selection process.
```

@stubbornella can you take a look?

documentation will be provided to guide proposers towards proposals
that meet the formal criteria (e.g. around testability) and which can
be assessed according to the agreed criteria.

Items from Interop 2025 which have not yet reached full
interoperability will be automatically resubmitted, and will go
through the process as for any other proposal. In addition new
proposals can be submitted that extend the scope of existing focus
areas (whether or not they have reached full interoperability).

As in previous years proposals can either be for focus areas or
investigations. For simplicity the remainder of this document will
only refer to focus area proposals, but this also covers proposals for
investigation efforts.

### Champion Selection

**Timeline**: October 9th, 16th, 23rd, 30th

Interop participants propose focus areas, consisting of one or more
proposals, that they would like to champion. Proposals will initially
be shared in time for the meeting on October 16th, and champions
finalised on October 23rd.

Where more than one participant wants to champion the same focus area
proposal a single champion must be nominated for the remainder of the
process.

> Champions and seconds (i.e. organizations that would champion a focus

Reviewer 2: Did we actually use the concept of "seconds" last year? I don't remember those getting assigned.
Unless we expect that concept to be useful this year, we should consider dropping it.

Author: Updated.

proposals, the participants are encouraged to work out a way to divide
the proposals into focus areas in a way that satisfies everyone. If
this is impossible then carrying forward multiple competing focus area
proposals is permitted, but final selections will only allow each
proposal to appear in a single focus area.

Any proposal without a champion will not be carried forward.

### Focus Area Proposal Refinement

**Timeline**: November 6th, November 13th

Champions gather evidence of the impact of their proposals, using the
rubric developed earlier. Specific participants may also be asked
to provide (publicly available) information for all proposals where
that participant has ownership of the information (e.g. standards
positions, use counter data).

### Focus Area Proposal Presentation

**Timeline**: November 20th (double-length meeting)

Each participant gets 15 minutes of meeting time to present the
proposals they are championing, and make the case that they should be
considered a priority for inclusion in Interop 2026.

### Focus Area Proposal Ranking

**Timeline**: Before December 4th

Participants rank each proposal as P1 (highest priority) to P3 (lowest

> priority), or veto.

Reviewer 2: ```suggestion
priority), or veto. Ranks are assigned to the individual proposals, not the overall focus areas.
```
This was a point of confusion for Interop2025 so we should try to clarify it here.

I could also live with assigning ranks to the overall focus areas, but I thought there was a lot of useful signal from folks having ranked individual proposals. It's easier to spot opportunities to move things around to make overall focus areas more viable.

Author: updated
